app/kubemci/pkg/gcp/healthcheck/healthchecksyncer_test.go

```diff
@@ -83,12 +102,22 @@ func TestEnsureHealthCheck(t *testing.T) {
 	}
 
 	for _, c := range testCases {
+		var svcPort int32 = 80
```

Comment:
svcPort := 80 will also work?

Reply:
I've changed it but still explicitly set it to `int32`, since `api_v1.ContainerPort` takes an `int32` and there's a TODO in `intstr.FromInt` to change it to take an `int32` as well, but currently we have to cast it to `int`